sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryFileLoadsSchemaTransformConfiguration.java

```diff
@@ -27,25 +27,25 @@
 /**
  * Configuration for writing to BigQuery.
  *
- * <p>This class is meant to be used with {@link BigQueryWriteSchemaTransformProvider}.
+ * <p>This class is meant to be used with {@link BigQueryFileLoadsWriteSchemaTransformProvider}.
  *
  * <p><b>Internal only:</b> This class is actively being worked on, and it will likely change. We
  * provide no backwards compatibility guarantees, and it should not be implemented outside the Beam
  * repository.
  */
 @DefaultSchema(AutoValueSchema.class)
 @AutoValue
-public abstract class BigQueryWriteSchemaTransformConfiguration {
+public abstract class BigQueryFileLoadsSchemaTransformConfiguration {
```

Comment:
wdyt of `BigQueryFileLoadsWriteSchemaTransformConfiguration`? better to keep provider and configuration name consistent